server/Dockerfile

```diff
@@ -58,4 +58,6 @@ ENV DEFAULT_CATTLE_MACHINE_EXECUTE true
 ENV DEFAULT_CATTLE_COMPOSE_EXECUTOR_EXECUTE true
 ENV DEFAULT_CATTLE_CATALOG_EXECUTE true
 
+RUN mkdir -p /var/run/mysqld && chmod -R 777 /var/run/mysqld
```

Comment:
Can you move this line to after the apt-get.  You can't have any modifying Dockerfile lines after a `VOLUME` because it screws up prepopulation of the data volume.  Yeah, it's a weird issue.